### PR TITLE
healthcare_service == type_of_care

### PR DESCRIPTION
**Pull request by jperk51**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
There's a bug on the Mobile FE where we are using healthcare_service field to backfill null ToC which shouldn't be done. Until the FE can fix that we'll code healthcare_service to be the same as ToC. 

## Related issue(s)

- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb [hunk 1]
```diff
@@ -98,36 +98,36 @@
       end
     end
 
-    context 'backfill clinic service returns data' do
```
Comment (kpethtel) [inline modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb:101]: FWIW, there are number of ways of disabling tests. For example, you can change put an x in front of the test block, like `xdescribe`, `xcontext`, or `xit`. Commenting them out is also fine for a temporary fix.

**Review by kpethtel — APPROVED** [review]
If I understand this correctly, the plan is to change the code back after the FE change? So when that happens, the current version of the mobile app will be correct but the older versions will revert to how they are now.

File: modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb (hunk 1 above)
Comment (jperk51) [reply modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb:101]: 👍

**Comment by jperk51** [comment]
> If I understand this correctly, the plan is to change the code back after the FE change? So when that happens, the current version of the mobile app will be correct but the older versions will revert to how they are now.

Not sure what we are going to do with this field, but we need to have a larger conversation around it, but for now we don't want users to see strings not intended for them. As for older versions of the app we may leave this in place until most people have updated, again, not 100% what we want to do with regards to that yet.

**Comment by jperk51** [comment]
Closing this because I think I should go about it a different way.

**Pull request closed without merging**